client/src/components/embedding/index.tsx

```diff
@@ -16,10 +16,124 @@ import actions from "../../actions";
 import { getDiscreteCellEmbeddingRowIndex } from "../../util/stateManager/viewStackHelpers";
 import { track } from "../../analytics";
 import { EVENTS } from "../../analytics/events";
-
 // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
 type EmbeddingState = any;
 
+const heatmapIcon = (
```

Comment:
@tihuan I was having trouble importing an svg file, so I temporarily added the svg component in-file. Declaring it in a `.d.ts` file and adding it to `tsconfig.json` did not help... any tips?

Reply:
Finally fixed everything I think lol

https://github.com/chanzuckerberg/single-cell-explorer/pull/391 PTAL thank you!